src/TextUI/Command.php

```diff
@@ -248,7 +250,7 @@ protected function handleArguments(array $argv)
         foreach ($this->options[0] as $option) {
             switch ($option[0]) {
                 case '--colors':
-                    $this->arguments['colors'] = $option[1] ?: PHPUnit_TextUI_ResultPrinter::COLOR_AUTO;
+                    $this->arguments['colors'] = $input->getOption('colors');
```

Comment:
@sebastianbergmann I implemented the parsing using symfony console for the `colors` option, so that you could see how its done. 

I'll work later on all the other options.